modules/nix/linux-builder.nix

```diff
@@ -77,6 +78,22 @@ in
         This sets the corresponding `nix.buildMachines.*.supportedFeatures` option.
       '';
     };
+
+    workingDirectory = mkOption {
+      type = types.str;
+      default = "/var/lib/darwin-builder";
+      description = lib.mdDoc ''
+        The working directory of the Linux builder daemon process.
+      '';
+    };
+
+    ephemeral = mkEnableOption (lib.mdDoc ''
+      delete the builder's disk image on ever service restart.
+
+      This is useful whenever you change the builder's system configuration or
+      simply don't want it to accumulate state. Leave it disabled if you need to
+      rely on secrets or other information in the builder's file system.
```

Comment:
```suggestion
      Wipe the builder's filesystem on every restart.

      This is disabled by default as maintaining the builder's Nix Store reduces
      rebuilds. You can enable this if you don't want your builder to accumulate
      state.
```

Reply:
btw the small first letter is due to the fact that `mkEnableOption` prefixes the doc string with `"Whether to "`, or is that disabled in nix-darwin?

Reply:
done